pkg/names/names.go

```diff
@@ -224,3 +224,6 @@ const DefaultClusterName = "default"
 
 // DashboardNamespace is the namespace where dashboards are created
 const DashboardNamespace = "openshift-config-managed"
+
+// DefaultUserAgent is the default UserAgent set on the clients
+const DefaultUserAgent = "cluster-network-operator"
```

Comment:
This is the same value as the fieldManager we've determined to be deprecated,  I don't think this will work in earlier releases where we don't plane to remove the deprecation cleanup logic.

Reply:
ok, would change it to `network-operator` then, so this makes stale fields which are managed by deprecated field manager (`cluster-network-operator`) automatically get removed when this version of code runs, am i right ?